scripts/huggingface/upload_model.py

```diff
@@ -0,0 +1,80 @@
+import inspect
+import logging
+import pathlib
+
+import hydra
+import torch
+from huggingface_hub import PyTorchModelHubMixin
+from hydra.utils import instantiate
+from omegaconf import DictConfig
+
+from the_well.data import WellDataModule
+
+logger = logging.getLogger("the_well")
+
+CONFIG_DIR = (
+    pathlib.Path(__file__) / ".." / ".." / ".." / "the_well" / "benchmark" / "configs"
+).resolve()
```

Comment:
```suggestion
CONFIG_DIR = pathlib.Path(__file__).resolve().parent.parent.parent / "the_well" / "benchmark" / "configs"
```

Reply:
Actually you know what, I kind of prefer `".."`... It's more intuitive to me. So, I take this back. Up to you!